gulpfile.js

```diff
@@ -349,8 +351,7 @@ gulp.task("release", function(cb) {
     "release:git:tag:release",
     // checkout a new branch funcpack-release-vx.x.x
     "release:git:checkout:funcpack",
-    // build and run funcpack
```

Comment:
check that the release process still works as `release:git:checkout:funcpack` may fail since now the tree is dirty (due to the build happening before)

Reply:
still works (all generated files are git-ignored)